spreadsheet/workbook.go

```diff
@@ -673,6 +682,12 @@ func (wb *Workbook) AddImage(i common.Image) (common.ImageRef, error) {
 	if i.Size.X == 0 || i.Size.Y == 0 {
 		return r, errors.New("image must have a valid size")
 	}
+	if i.Path != "" {
+		err := tempstorage.Add(i.Path)
```

Comment:
Need to close/remove this eventually also, clean up memory.

Reply:
added close/finazliers to spreadsheet, document and presentation